packs/classfeatures/unfurling-brocade.json

```diff
@@ -24,7 +24,49 @@
             "remaster": true,
             "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Magus.UnfurlingBrocade.QiInfusedFabric.Label",
+                "option": "qi-infused-fabric",
+                "suboptions": [
+                    {
+                        "label": "PF2E.TraitHeldTwoHands",
+                        "value": "2"
+                    },
+                    {
+                        "label": "PF2E.TraitHeldOneHand",
+                        "value": "1"
+                    }
+                ],
```

Comment:
Can we not just use `item:hands-held:x`?

Reply:
You mean using that as the roll option in this rule element? Or using the automatically generated `item:hands-held:x` from wielding a weapon in one hand? If it's the latter, the problem is that since the Bladed Scarf can't normally be used in one hand, the character sheet won't let you use it.

Reply:
Rather than hacking this together with a weird toggle we should request usage alterations then.

Reply:
I did add it to the wish list, albeit bounded to weapons rather than usage in general. I can broaden the request, and I guess just get rid of this part of the automation.